test_workloads.py

```diff
@@ -234,7 +234,8 @@ def test_helloworld_workload(self):
         assert("Hello, world" in helloworld_contents)
 
     @pytest.mark.examples
-    @pytest.mark.skipif((base_os not in ["ubuntu20.04", "ubuntu22.04", "debian11"]) or ((int(no_cores) < 16) and sgx_mode == '1'),
+    @pytest.mark.skipif((base_os not in ["ubuntu20.04", "ubuntu22.04", "debian11"]) or (base_os == "ubuntu24.04")
```

Comment:
base_os check for ubuntu 24.04 is not required here

Reply:
Modified